redis/redis/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '2'
+services:
+  gateway:
+    build: gateway
+    ports:
+      - "8000:8000"
+    hostname: "kaazing.example.com"
+    networks: 
+      tut-redis: 
+        ipv4_address: 172.19.0.3
+    volumes:
+    - /var/log/tut-redis/gateway:/kaazing-gateway/log
+  redis:
+    build: broker
+    ports:
+      - "6379:6379"
+    networks: 
+      tut-redis: 
+        ipv4_address: 172.19.0.4      
+    volumes:
+     - /var/log/tut-redis/redis:/data/log/
+networks:
+  tut-redis:
```

Comment:
My philosophy on creating this is to keep everything bare minimal unless absolutely needed, with that stated I don't see a reason for the network config and assigning ip addresses, was this added for a specific reason?